### DISCUSSION TITLE
Port could not be cast to integer value

### DISCUSSION THREAD

**Discussion by AndrewOYLK** [apache/airflow#21148] [Q&A]
Redis has been started locally, and the airflow.cfg configuration has been changed，but I get an error when I run the celery worker, I need help explaining why，Thanks

![image](https://user-images.githubusercontent.com/19611582/151330330-846213d8-c269-4303-a0e7-bc8f5a9f8cdd.png)

the command: "airflow celery worker"
![image](https://user-images.githubusercontent.com/19611582/151330099-13766c88-9819-4920-a980-e930353ed8f3.png)

**Comment by potiuk** [comment]
Just look for it - somewhere in your configuration you will find `Oufva` - most likely typo you made in the configuration. But where it is, you are the only one to know :)

**Comment by asasisekar** [comment]
If passwords has any of /, ?, #, @, or : characters in the connection url string, will raise a ValueError while urllib parse it. 
If the URL / password is decomposed before parsing like below, no error will be raised.

>>> import urllib.parse
>>> urllib.parse.quote('aa/bb?cc#dd@ee:ff', safe='')
'aa%2Fbb%3Fcc%23dd%40ee%3Aff'

**Reply by potiuk** [reply]
Yes. It's written in our docs about connection. There is a even a tool to generate the properly encoded URL for connection.
